ui/components/app/snaps/snap-install-warning/snap-install-warning.js

```diff
@@ -75,11 +75,26 @@ export default function SnapInstallWarning({
       const secondWarningSubject = permissionWarnings[1].warningMessageSubject;
       return [
         <Text
-          fontWeight={FontWeight.Medium}
+          fontWeight={FontWeight.Normal}
           as="span"
           key="warningMessageSubject"
         >
-          {t('andForTwoItems', [firstWarningSubject, secondWarningSubject])}
+          {t('andForTwoItems', [
+            <Text
+              fontWeight={FontWeight.Medium}
+              as="span"
+              key={`${firstWarningSubject}_and_first`}
+            >
```

Comment:
Prefer using the inherit variant rather than setting it as a span directly

```suggestion
            <Text
              fontWeight={FontWeight.Medium}
              variant={TextVariant.inherit}
              key={`${firstWarningSubject}_and_first`}
            >
```

Reply:
Updated.